packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -981,6 +992,9 @@ class SliverAppBar extends StatefulWidget {
     this.stretchTriggerOffset = 100.0,
     this.onStretchTrigger,
     this.shape,
+    this.preventShowOnScreenUnpinning = true,
+    this.minShowOnScreenExtent,
```

Comment:
These two (maybe all three) new properties seem to be on a different abstraction level then the other properties.